shared/src/packages/drivers/libs/types/driver-entity.type.ts

```diff
@@ -6,6 +6,8 @@ type DriverEntity = {
   driverLicenseNumber: string;
   userId: UserEntityT['id'];
   businessId: BusinessEntityT['id'];
+  createdAt?: Date;
```

Comment:
why it's optional? 
also rewrite it to string type I guess, as through network there won't be type Date